webapp/app/js/perf.js

```diff
@@ -591,7 +678,7 @@ perf.config(function($stateProvider, $urlRouterProvider) {
 
   $stateProvider.state('graphs', {
     templateUrl: 'partials/perf/perfctrl.html',
-    url: '/graphs?timerange&series&highlightedRevision',
+    url: '/graphs?timerange&series&highlightedRevision&zoomLevel',
```

Comment:
zoomLevel implies a single integer implying how "zoomed in" we are. Let's just call this variable "zoom" everywhere (in the url, in the code, etc.)